modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavascriptClientCodegen.java

```diff
@@ -289,14 +298,35 @@ public String escapeReservedWord(String name) {
         return "_" + name;
     }
 
+    private String createPath(String... segments) {
```

Comment:
Do you mind elaborating a bit more on what `createPath` does? A docstring to explain its usage would be ideal.